### ISSUE TITLE
Error: plugin:lodash/recommended: Configuration for rule "lodash/prefer-flat-map" is invalid:

### ISSUE THREAD

**Issue by godu** [wix/eslint-plugin-lodash#42]
In `lodash/recommended`, `lodash/prefer-flat-map` is a string value instead of number one. https://github.com/wix/eslint-plugin-lodash/blob/master/index.js#L31
```Shell
Error: plugin:lodash/recommended:
        Configuration for rule "lodash/prefer-flat-map" is invalid:
        Severity should be one of the following: 0 = off, 1 = warning, 2 = error (you passed "2").
```

**Comment by ganimomer** [comment]
Merged in `1.4.1`. Thanks for the PR!
